### PR TITLE
Slight changes to APOSMM tutorial

### PR DESCRIPTION
**Pull request by jmlarson1**

### CODE REVIEW

File: docs/tutorials/aposmm_tutorial.rst [hunk 1]
```diff
@@ -56,29 +56,31 @@ APOSMM Operations
...
-by passing them back to be evaluated by the simulation routine. If so, new local
-optimization runs are started from those points. This continues until each run
-converges to a minimum:
+Specifically, APOSMM will begin local optimization runs from evaluated points that
+don't have points with smaller function values nearby (within a threshold
+``r_k``). For the above example, after APOSMM receives the evaluations of the
+uniformly sampled points, it will begin at most ``max_active_runs``  local
+optimization runs.
+
+Each local optimization run provides a new point to APOSMM, which gives them to
```
Comment (shuds13) [inline docs/tutorials/aposmm_tutorial.rst:78]: Maybe add something like:
At each iteration, ....
Entirely optional, it just sounds a bit to me like the optimization run is completed when it provides a point.
And/or where it says "as runs complete", perhaps something like (when a mininum is found).
Just my reading of it.

File: libensemble/tests/scaling_tests/warpx/readme.txt [hunk 2]
```diff
@@ -13,14 +15,21 @@ res2          64  4096      54s       4.557730935993e-06
...
-- The jsrun command should be modified as, e.g., "jsrun ... warpx.exe inputs amr.n_cell=128 8192" for resolution res3
+- The timeout in the batch script should be larger for higher resolutions (see
+  numbers above)
+- The jsrun command should be modified as, e.g., "jsrun ... warpx.exe inputs
+  amr.n_cell=128 8192" for resolution res3
 
 Note that this test shows a nice converge for a case with relatively high emittance.
-In the optimization process, some runs will have a luwer emittance (potentially 10x), and they may require a higher resolution to reach the same level of convergence.
-Let us keep this in mind, and maybe add a res5 to the list above. This high-resolution one should also fit on 1 GPU, but it will be worth testing.
+In the optimization process, some runs will have a luwer emittance (potentially
```
Comment (shuds13) [inline libensemble/tests/scaling_tests/warpx/readme.txt:31]: lower?

Comment (jmlarson1) [reply libensemble/tests/scaling_tests/warpx/readme.txt]: Yes.

File: docs/tutorials/aposmm_tutorial.rst (hunk 1 above)
Comment (jmlarson1) [reply docs/tutorials/aposmm_tutorial.rst]: Thanks. I have iterated on your suggestions and hopefully clarified this. Does it read better now?

Comment (shuds13) [reply docs/tutorials/aposmm_tutorial.rst]: Yes I think this is clearer.

**Comment by shuds13** [comment]
@jmlarson1 @jlnav This is still unmerged (into docs/scipy2020). Do we want to merge it, and then to develop? Or wait till after scipy?

**Pull request merged**